spark-plugins/docs/DecisionTreeTrainer-sparksink.md

```diff
@@ -0,0 +1,50 @@
+# DecisionTreeTrainer
+
+
+Description
+-----------
+Trains a regression model based upon a particular label and featuresToInclude of a record. Saves this model to a FileSet.
+
+Use Case
+--------
+This sink can be used when you have a sample data and you want to use it to build a Decision Tree Regression model.
+
+Properties
+----------
+**fileSetName:** The name of the FileSet to save the model to.
+
+**path:** Path of the FileSet to save the model to.
+
+**featuresToInclude:** A comma-separated sequence of fields to use for training. If empty, all fields will be
+considered for training. Features to be used, must be from one of the following type: int, long, float or double. Both
+*featuresToInclude* and *featuresToExclude* fields cannot be specified.
+
+**featuresToExclude:** A comma-separated sequence of fields to be excluded when training. If empty, all the fields will
+be considered for training.  Both *featuresToInclude* and *featuresToExclude* fields cannot be specified.
+
+**labelField:** The field from which to get the prediction. It must be of type double.
+
+**maxDepth:** Maximum depth of the tree.
+              For example, depth 0 means 1 leaf node; depth 1 means 1 internal node + 2 leaf nodes. Default is 10.
```

Comment:
Should use consistent indentation. Other fields have this all the way to the left.